docs/tutorials/using-backstage-proxy-within-plugin.md

```diff
@@ -0,0 +1,210 @@
+---
+id: using-backstage-proxy-within-plugin
+title: Using the Backstage Proxy from Within a Plugin
+# prettier-ignore
+description: Guide on how to create a set of API bindings that interface with a backend via the backstage proxy
+---
+
+This guide walks you through setting up a simple proxy to an existing API that
+is deployed externally to backstage and sending requests to that API from within
+a backstage frontend plugin. For a more detailed description of the APIs used
+here, check out the [Utility APIs section](../api/utility-apis.md).
+
+If your plugin requires access to an API, backstage offers
+[3 options](../plugins/call-existing-api.md):
+
+1. you can
+   [access the API directly](../plugins/call-existing-api.md#issuing-requests-directly),
+1. you can create a [backend plugin](../plugins/backend-plugin.md) if you are
+   implementing the API alongside your frontend plugin
+1. you can configure backstage to proxy to an already existing API. The third
```

Comment:
Gotcha, happy to make that change. Ideally this tutorial is as useful as possible. What are your thoughts on a minor restructure that makes it a thread through creating a utility api, with a few notes about proxying?

Reply:
I think it's best to have any more in-depth stuff around utility APIs in a different place in the docs, since it'll be used in a bunch of other sections too. So with that I think we should link out to the Utility API docs for that part, but also happy to look at any suggested improvements or filling in gaps in that section too :grin: